HW2/P5/driver.py

```diff
@@ -80,3 +82,19 @@ def randcolor():
             # SUBPROBLEM 3: sort objects by location.  Be sure to update the
             # grid if objects' indices change!  Also be sure to sort the
             # velocities with their object positions!
+            
+            # get d values from Hilbert curve function
+            # multiply positions by a large power of 2 because hilbert function requires integers
+            # and a power of 2 as arguments
+            get_ds_from_positions(256, (positions*256).astype(np.uint32), ds_array, num_balls)
```

Comment:
Just curious, how does Hilbert ordering work? I did mine with Morton sort so didn't read about this.